crates/ott-collector/Cargo.toml

```diff
@@ -8,9 +8,11 @@ edition = "2021"
 anyhow.workspace = true
 ott-common.workspace = true
 reqwest.workspace = true
-rocket.workspace = true
 serde.workspace = true
 serde_json.workspace = true
 tokio.workspace = true
 tracing.workspace = true
 tracing-subscriber.workspace = true
+
+[dependencies.rocket]
+features = ["json"]
```

Comment:
nit: do this in the root `Cargo.toml`, in one line. eg `rocket = {version: ... , features = ["json"]}` like the other deps

Reply:
This is not resolved. Move the dependency declaration to the workspace Cargo.toml, in the root of the repo. The dependency declaration in this file should remain unchanged as `rocket.workspace = true`

Reply:
> This is not resolved. Move the dependency declaration to the workspace Cargo.toml, in the root of the repo. The dependency declaration in this file should remain unchanged as `rocket.workspace = true`

Huh I thought I changed this. Maybe it didn't get added to the commit for whatever reason